lib/expeditor/command.rb

```diff
@@ -37,18 +45,18 @@ def start_with_retry(retryable_options = {})
     end
 
     def started?
-      @normal_future.executed?
+      !!@normal_future && @normal_future.executed?
```

Comment:
I think it's not good. Because `@normal_future` should be hidden from library users.
I follow the style guide if it exist.